src/main/java/org/folio/circulation/domain/policy/Period.java

```diff
@@ -200,4 +202,12 @@ public boolean isEqualToDateTillNow(DateTime startDate) {
 
     return now.isEqual(startPlusPeriod);
   }
+
+  public boolean isZeroDuration() {
```

Comment:
That was a poor suggestion on my part. I should've suggested, `hasZeroDuration` apologies :-/ 

```suggestion
  public boolean hasZeroDuration() {
```